boards/lolin_s3.json

```diff
@@ -15,6 +15,7 @@
     "f_cpu": "240000000L",
     "f_flash": "80000000L",
     "flash_mode": "qio",
+    "psram_type": "opi",
```

Comment:
Isn't the PSRAM type already covered in just above in the memory type field? Where do you use it?
```json
{
  "build": {
    "arduino": {
      "ldscript": "esp32s3_out.ld",
      "partitions": "default_16MB.csv",
      "memory_type": "qio_opi"
    },
...
}
```

Reply:
My fault, did not look into /builder/main.py (def _get_board_memory_type(env)), removed duplicate.